src/packages/builder/src/vite-config.ts

```diff
@@ -19,6 +20,15 @@ export const viteConfig = ({
 	backendUrl,
 	base = '/',
 }: ViteConfigOptions): InlineConfig => {
+	const optimizeDeps = [
```

Comment:
It'd be great to put a comment here explaining why these are here, e.g. linking to the GitHub Issue etc.